src/main/java/org/springframework/data/elasticsearch/config/ElasticsearchConfigurationSupport.java

```diff
@@ -43,14 +42,16 @@
 
 /**
  * @author Christoph Strobl
+ * @author Peter-Josef Meisch
  * @since 3.2
  */
 @Configuration
 public class ElasticsearchConfigurationSupport {
 
 	@Bean
-	public ElasticsearchConverter elasticsearchConverter() {
-		return new MappingElasticsearchConverter(elasticsearchMappingContext());
+	public MappingElasticsearchConverter elasticsearchEntityMapper(
+			SimpleElasticsearchMappingContext elasticsearchMappingContext) {
+		return new MappingElasticsearchConverter(elasticsearchMappingContext, new DefaultConversionService());
```

Comment:
Let's use a constructor without a `conversionService` here to align with other modules.